### PR TITLE
Reduce Unchecked Code in Benchmark Conversions

### PR DESCRIPTION
**Pull request by lenary**
This is the changes for #42.

### CODE REVIEW

**Review by dtarditi — APPROVED** [review]
Looks good - thanks!

File: MultiSource/Benchmarks/Olden/power/main.c [hunk 1]
```diff
@@ -61,12 +61,13 @@ _Unchecked int main(int argc, _Array_ptr<char*> argv : count(argc))
   double d_theta_R,d_theta_I;
```
Comment (dtarditi) [inline MultiSource/Benchmarks/Olden/power/main.c:61]: You could make the body of main a checked  scope by placing _Checked before the body of the function:
```
_Unchecked int main(int argc, ...) 
_Checked {
 ...
}
```

**Comment by dtarditi** [comment]
I added one suggestion.

File: MultiSource/Benchmarks/Olden/power/main.c (hunk 1 above)
Comment (lenary) [reply MultiSource/Benchmarks/Olden/power/main.c:61]: Will do

**Pull request merged**